lectures/markov_chains_II.md

```diff
@@ -228,9 +228,9 @@ distribution, then, for all $x \in S$,
 
 Here
 
-* $\{X_t\}$ is a Markov chain with stochastic matrix $P$ and initial
+* $\{X_t\}$ is a Markov chain with stochastic matrix $P$ and initial.
   distribution $\psi_0$
-* $\mathbf{1}\{X_t = x\} = 1$ if $X_t = x$ and zero otherwise
+* $1 \cdot \{X_t = x\} = 1$ if $X_t = x$ and zero otherwise.
```

Comment:
The original is correct --- this is an indicator function.

Please revert to the original but then change `\mathbf` to `\mathbb`.

Reply:
Should be same question as above.

Reply:
@HengchengZhang , these are different questions.

Let's agree that $\mathbb 1$ on its own indicates a vector or matrix of ones.

In addition, the symbol $\mathbb 1\{ \cdot \}$ is an indicator function.  For example,

$$ \mathbb 1\{ x \in X \}  = 1 $$

if $x \in A$ and zero otherwise.